main/main.cpp

```diff
@@ -2187,8 +2205,11 @@ bool Main::iteration() {
 			break;
 		}
 
+		NavigationServer::get_singleton_mut()->process(frame_slice * time_scale);
 		message_queue->flush();
 
+		NavigationServer::get_singleton_mut()->process(frame_slice * time_scale);
```

Comment:
Is it meant to be processed twice per iteration? That's not the case in the `master` branch.